sql/Votes_pre.sql

```diff
@@ -1,7 +1,7 @@
 DROP TABLE IF EXISTS Votes CASCADE;
 CREATE TABLE Votes (
    Id                int         PRIMARY KEY ,
-   PostId            int         not NULL    ,
+   PostId            int         , -- not NULL    ,
```

Comment:
I'd rather not add the foreign key constraints if they require dropping the `not NULL` on these fields.

One will not be updating the tables anyway after they have been created once. So the primary reason to have foreign key constraints is for performance (to make `EXISTS` queries faster, at least), correct?

In that case, I'd rather leave them up to the discretion of the user while preserving as many 'sanity' checks as we can (e.g. not NULL constraints) which are provided by the underlying real-data.